ush/forecast_postdet.sh

```diff
@@ -361,6 +361,15 @@ WW3_postdet() {
     echo "WW3 will start from rest!"
   fi  # [[ "${warm_start}" == ".true." ]]
 
+  # Link restart files
+  local restart_file
+  # Use restart_date if it was determined above, otherwise use initialization date
+  for (( vdate = "${restart_date:-${PDY}${cyc}}";  vdate <= forecast_end_cycle;
+         vdate = $(date --utc -d "${vdate:0:8} ${vdate:8:2} + ${restart_interval} hours" +%Y%m%d%H) )); do
+    restart_file="${vdate:0:8}.${vdate:8:2}0000.restart.ww3"
+    ${NLN} "${DATArestart}/WW3_RESTART/${restart_file}" "${restart_file}"
+  done
```

Comment:
Is this correct? Why is linking restarts from restart date all the way to end of the forecast needed here for initialization?
Only the restarts for the `restart_date` that was determined is needed for initialization. 
There should be no loop here for initialization.

A loop is needed for linking the restarts from the `next` restart time to the end forecast cycle for restart capability.

Reply:
There's already a separate part to copy the initial restart in, if necessary. Just need to adjust the start date for the links when that happens.